certified-connectors/PIPware KPIs/README.md

```diff
@@ -0,0 +1,30 @@
+
+
+## PIPware Connector
+This connector allows you to save KPI Target or KPI Actual data to your PIPware instance. This is more efficient than capturing the data into PIPware manually. Commonly used data sources are Excel or SQL, but any source returning data in the expected format can be used.
+
+## Pre-requisites
+You will need the following to proceed:
+* A Power Automate plan
```

Comment:
Could this please be changed to say "A Power Platform premium plan"? Thanks!

Reply:
Hi @dougelliott  I have made the change